src/coord/src/persistcfg.rs

```diff
@@ -311,6 +325,44 @@ impl PersisterWithConfig {
     }
 }
 
+fn stream_desc_from_name(
+    name: String,
+    persister: &RuntimeClient,
+) -> Result<PersistStreamDesc, Error> {
+    let description = persister.get_description(&name);
+    let description = match description {
+        Ok(description) => description,
+        Err(Error::UnknownRegistration(error_name)) if name == error_name => {
+            // The stream has not been created yet, so return the initial seal timestamp of
+            // streams.
+            // TODO: We might want to codify this somewhere?
+            use timely::progress::Timestamp;
```

Comment:
nit: group this with the imports at the top of the file please

Reply:
I must have picked this up from the timely/differential code base... 🙈